### DISCUSSION TITLE
enhancement: HPcost and MPcost in spells using percentages and Hpdamage, Mpdamage

### DISCUSSION THREAD

**Discussion by cydyn** [AscensionGameDev/Intersect-Engine#2205] [Ideas]
### Description

In spells, it would be useful to have the option to use manacost and hpcost percentages instead of just integers.
Thanks to this, we could create spells that take a given amount of our maximum mana or hp.
Additionally, if there is an option, add the Hpdamage and Mpdamage percentage option

### Intended Use-Case

The only thing is that this option will be useful when we want the spell to take more than a given fixed amount.
This is useful when there is a lot of mana, for example, and spells take little

### Duplicate Check

- [X] This enhancement idea is not a duplicate to the best of my knowledge.

**Comment by PyroTech03** [comment]
I'd love to see % health/max health type of damage options as it's a common thing. Cost not as common but I have seen it and could still be useful.
